configurations/perf/regression-nosqlbench-aws.yaml

```diff
@@ -0,0 +1,6 @@
+test_duration: 360
```

Comment:
Move this out from the configuration to the test yaml (unless it differs per config).

Reply:
Fixed